src/cli/Show.cpp

```diff
@@ -96,6 +96,8 @@ int Show::executeWithDatabase(QSharedPointer<Database> database, QSharedPointer<
         }
         if (entry->attributes()->isProtected(attributeName) && showAttributeNames && !showProtectedAttributes) {
             outputTextStream << "PROTECTED" << endl;
+        } else if (attributeName.compare("Uuid") == 0) {
+            outputTextStream << entry->uuid().toString().remove((QRegExp("[{}]"))) << endl;
```

Comment:
Use `entry->uuid().toString(QUuid::WithoutBraces)` (`QRegExp` is deprecated BTW).

Reply:
Got it, thanks.